react/Gallery.js

```diff
@@ -13,13 +14,32 @@ import searchResult from './searchResult.css'
 /**
  * Canonical gallery that displays a list of given products.
  */
+
+const TWO_ITEMS = 2
+const ONE_ITEM = 1
+
 class Gallery extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { prevLayoutMode: this.props.layoutMode }
```

Comment:
I think the use of the constructor here is pointless, you can set the state propertly more directly, what do you think?